arch/PKGBUILD

```diff
@@ -0,0 +1,70 @@
+# adapted from https://github.com/archlinux/svntogit-packages/blob/packages/varnish/trunk/PKGBUILD
+
+# before building, be sure to:
+# - replace @VERSION@ with the proper Varnish version to use
+# - run `makepkg -fg >> PKGBUILD` to generate the checksum entries
+
+pkgname=varnish
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="High-performance HTTP accelerator"
+arch=('x86_64')
+url="https://www.varnish-cache.org/"
+license=('BSD')
+depends=('gcc' 'libnsl' 'pcre2')
+makedepends=('python-docutils' 'python-sphinx' 'git')
```

Comment:
Do we really need git here?

Reply:
good catch, we actually don't. This was ported from the official `arch` repos that use this `git` repo for their `systemd` files

Reply:
Can you also try removing docutils and sphinx to only makedepend on python 3?

Reply:
without `python-sphinx`:
```
checking for sphinx-build-3.6... no
checking for sphinx-build-3... no
checking for sphinx-build... no
configure: error: sphinx-build is needed to build Varnish, please install python3-sphinx.
```

Reply:
and actually, if I remove both, `configure` complains about `rst2man`, provided by `python-docutils`, so we also need it explicitly, and not just as a dependency to `python-sphinx`

Reply:
I forgot that we poorly handle sphinx and rst2man, my bad.